### ISSUE TITLE
Can't Start Espanso 

### ISSUE THREAD

**Issue by lifesign** [federico-terzi/espanso#434]
Upgrade Espanso to version 0.7.1, got this error when starting.

➜ espanso start

Error starting launchd daemon with status: exit code: 3

macOS: 10.15.6

**Comment by federico-terzi** [comment]
@lifesign Hey, Have you called `espanso register` beforehand?
You should repeat the procedure at each update: https://espanso.org/install/mac/#enabling-accessibility

**Comment by lifesign** [comment]
🤔. forgot this step, thx~ it's ok now

**Comment by federico-terzi** [comment]
Glad you sorted that out :)

**Comment by gabrielvincent** [comment]
I had this same problem and was able to run Espanso by doing `espanso register` before `espanso start`. But that step is not listed in [the Getting Started section of the site](https://espanso.org/docs/get-started/). After I found this issue I went back to see if I missed a step in the getting started or in the installation walkthroughs. I found the Enable Accessibility at the bottom of the page at https://espanso.org/install/mac/#enabling-accessibility. Maybe it would be helpful to add `espanso register` as a command after `espanso --version` both in the HomeBrew and Manual Installation instructions.
